Sources/SwiftGraphQLClient/Extensions/Publishers+Extensions.swift

```diff
@@ -8,6 +8,42 @@ extension Publisher {
     func takeUntil<Terminator: Publisher>(_ terminator: Terminator) -> Publishers.TakenUntilPublisher<Self, Terminator> {
         Publishers.TakenUntilPublisher<Self, Terminator>(upstream: self, terminator: terminator)
     }
+
+    /// Takes the first emitted value and completes or throws an error
+    func first() async throws -> Output {
```

Comment:
`Publisher.values` only available starting from `iOS 15` and I was forced to cook up a custom implementation via continuations (`Publisher.first`)

Reply:
For clarity, I think it makes sense to keep these tests since they seem nicely packed and concise.

Reply:
perhaps it has changed since PR was created, as of now `swift-graphql` requires `.iOS(.v15),`